pkgs/top-level/all-packages.nix

```diff
@@ -3327,6 +3327,8 @@ with pkgs;
 
   async-profiler = callPackage ../development/tools/async-profiler { };
 
+  athens = callPackage ../servers/athens { };
```

Comment:
You should add this to `pkgs/by-name` instead. See https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md